app/mailers/review_confirmation.rb

```diff
@@ -0,0 +1,11 @@
+class ReviewConfirmation < ApplicationMailer
+	default from: 'michael.foster@outlook.com',
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Tab detected.